repos/newtonsoft-json.proj

```diff
@@ -12,7 +12,7 @@
     <NewtonsoftJsonKeyFilePath>$(KeysDir)Newtonsoft.Json.snk</NewtonsoftJsonKeyFilePath>
     <NewtonsoftJsonDirectory>$(ProjectDirectory)/Src/Newtonsoft.Json/</NewtonsoftJsonDirectory>
     <NewtonsoftJsonProjectPath>$(NewtonsoftJsonDirectory)Newtonsoft.Json.csproj</NewtonsoftJsonProjectPath>
-    <DotnetToolCommandArguments>"/p:DotnetOnly=true" "/p:Configuration=$(Configuration)" "/p:AssemblyOriginatorKeyFile=$(NewtonsoftJsonKeyFilePath)" "/p:SignAssembly=true" "/p:PublicSign=true" "/p:TreatWarningsAsErrors=false" "/p:AdditionalConstants=SIGNED"</DotnetToolCommandArguments>
+    <DotnetToolCommandArguments>"/p:DotnetOnly=true" "/p:Configuration=$(Configuration)" "/p:AssemblyOriginatorKeyFile=$(NewtonsoftJsonKeyFilePath)" "/p:SignAssembly=true" "/p:DelaySign=$(DelaySign)" "/p:PublicSign=true" "/p:TreatWarningsAsErrors=false" "/p:AdditionalConstants=SIGNED"</DotnetToolCommandArguments>
```

Comment:
I also tried `"/p:PublicSign=$(PublicSign)"` but that results in a runtime value of `/p:PublicSign=` being passed, which doesn't work. Any idea why `PublicSign` is undefined at this point?

Reply:
DelaySign is also set by MSBuild's common targets in MS.Common.CurrentVersion.targets: https://github.com/dotnet/msbuild/blob/1a6d753a7648baf0cc991af0c0d254a41d36eba6/src/Tasks/Microsoft.Common.CurrentVersion.targets#L567.  PublicSign is not, so I think this is expected.  We're passing in the SDK's default value for for `DelaySign` here instead of Arcade's due to this, but I think that's okay in this case since the rules for what we want to set it to are the same in both cases.